spec/lib/register_entry_spec.rb

```diff
@@ -41,4 +41,19 @@
       end
     end
   end
+
+  describe '#ordering' do
+    context 'given no arguments' do
+      it 'sets the ordering to the default' do
+        entry.order.should eq('id')
+      end
+    end
+
+    context 'given ordering arguments' do
+      it 'sets the ordering to the passed ordering' do
+        entry.order :example
+        entry.order.should eq(:example)
+      end
+    end
```

Comment:
I think we should have an integration test with e.g.

```
{ id: 1, name: 'Zeke' },
{ id: 2, name: 'Aaron' }
```
and check that the output is in the correct order for both default and when ordering is `name`.